portfolio/src/main/webapp/script.js

```diff
@@ -94,4 +94,11 @@ function showSlides(index) {
   }
   slides[slideIndex-1].style.display = 'block';
   dots[slideIndex-1].className += ' dot-active';
+}
+
+/** Fetch message from /data and push to index.html */
+async function getMessage() {
```

Comment:
nit: maybe something more descriptive for the function name like getFruitFact. Don't forget to also change it in the html file

Reply:
Gotcha, I changed the function name and the div tag's id name. Thanks Chris!